opentmi_client/opentmi_client.py

```diff
@@ -210,6 +217,8 @@ def __get_campaigns(self):
         return self.__get_JSON(url)
 
     def __get_suite(self, suite, options=''):
+        suite = suite if isinstance(suite, basestring) else str(suite)
```

Comment:
Agreed, since at the moment it is only called by get_suite that does not send None values forward, I see little to no advantage in keeping this problematic line in the function.

Removing the line for now.